<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWidget</class>
 <widget class="QFrame" name="StatisticsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="PickPeriodWidget" name="widgetPickPeriod" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>470</width>
       <height>80</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DailyTimelineGraph" name="dailyTimelineGraph">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1021</width>
       <height>241</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>1011</width>
       <height>311</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>311</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1041</width>
        <height>293</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,4,5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>.Helvetica Neue Desk UI</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Best Workday</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelBestWorkdayName">
           <property name="font">
            <font>
             <family>Lucida Sans Unicode</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>workdayNamePlaceholder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelBestWorkdayMsg">
           <property name="font">
            <font>
             <family>.Helvetica Neue Desk UI</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Average comparison message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="BarChart" name="workdayBarChart" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>249</width>
             <height>192</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="DistributionDiagram" name="topTagDiagram" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutBestWorktime" stretch="0,0">
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutBestWorktime">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>.Helvetica Neue Desk UI</family>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Best Worktime</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelBestWorktimeName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>18</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>.Helvetica Neue Desk UI</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Afternoon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelBestWorktimeHours">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>.Helvetica Neue Desk UI</family>
               <pointsize>11</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>14:00-18:00</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="TimeDiagram" name="timeDiagram" native="true"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PickPeriodWidget</class>
   <extends>QWidget</extends>
   <header>src/widgets/pickperiodwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DistributionDiagram</class>
   <extends>QWidget</extends>
   <header>src/widgets/DistributionDiagram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BarChart</class>
   <extends>QWidget</extends>
   <header>src/widgets/barchart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeDiagram</class>
   <extends>QWidget</extends>
   <header>widgets/timediagram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DailyTimelineGraph</class>
   <extends>QFrame</extends>
   <header>widgets/DailyTimelineGraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
